Formula/petsc.rb

```diff
@@ -0,0 +1,44 @@
+class Petsc < Formula
+  desc "Scalable solution of models that use partial differential equations"
+  homepage "https://www.mcs.anl.gov/petsc/index.html"
```

Comment:
http://www.mcs.anl.gov/petsc/

Reply:
Shouldn't we keep HTTPS?

Reply:
I assume the issue is the superfluous `index.html`.

Reply:
right